relay-server/src/actors/envelopes.rs

```diff
@@ -551,7 +551,7 @@ fn extract_session_metrics(session: &SessionUpdate, target: &mut Vec<Metric>) {
                 unit: MetricUnit::Duration(DurationPrecision::Second),
                 value: MetricValue::Distribution(duration),
                 timestamp,
-                tags,
+                tags: with_tag(&tags, "session.status", session.status),
```

Comment:
@jan-auer Do you remember if there was any particular reason we did not add this in the first place? `session.duration` can definitely be grouped by `session.status` in [sessions_v2](https://docs.sentry.io/api/releases/retrieve-release-health-session-statistics/).